smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -1822,7 +1822,7 @@ impl OwnedParameter {
 
 /// Check whether the given string is a valid contract entrypoint name.
 /// This is the case if and only if
-/// - the string is no more than [constants::MAX_FUNC_NAME_SIZE][m] bytes
+/// - the string is less than [constants::MAX_FUNC_NAME_SIZE][m] bytes
```

Comment:
🥇

Reply:
We sometimes check the byte length and sometimes the string length in this file. Based on the comment `MAX_FUNC_NAME_SIZE` should be the length in byte characters rather than string characters.

```
        if name.len() > constants::MAX_FUNC_NAME_SIZE {
            return Err(NewReceiveNameError::TooLong);
        }
```
```
    if name.as_bytes().len() >= constants::MAX_FUNC_NAME_SIZE {
        return Err(NewReceiveNameError::TooLong);
    }
```